assets/js/socket.js

```diff
@@ -52,12 +52,17 @@ let socket = new Socket("/socket", {params: {token: window.userToken}})
 //     end
 //
 // Finally, connect to the socket:
-socket.connect()
+socket.connect();
 
 // Now that you are connected, you can join channels with a topic:
-let channel = socket.channel("topic:subtopic", {})
-channel.join()
-  .receive("ok", resp => { console.log("Joined successfully", resp) })
-  .receive("error", resp => { console.log("Unable to join", resp) })
+let channel = socket.channel('band:lobby', {});
+channel
+  .join()
+  .receive('ok', resp => {
+    console.log('Joined successfully', resp);
+  })
+  .receive('error', resp => {
+    console.log('Unable to join', resp);
+  });
```

Comment:
We can actually comment this out. We've already got this functionality in band.ts